clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkClient.java

```diff
@@ -272,6 +272,7 @@ private boolean trySend(long now) {
 
     private void clientPoll(long timeout, long now) {
         client.poll(timeout, now);
+        now = time.milliseconds();
```

Comment:
It looks like the `now` parameter to clearUnsentRequests isn't actually used. We could just remove that parameter and not update `now` here.